r2/r2/lib/template_helpers.py

```diff
@@ -607,11 +607,11 @@ def simplified_timesince(date, include_tense=True):
     if date > timeago("1 minute"):
         return _("just now")
 
-    since = []
-    since.append(timesince(date))
+    since = timesince(date)
     if include_tense:
-        since.append(_("ago"))
-    return " ".join(since)
+        return _("%s ago") % since
```

Comment:
"%s ago" is already translated (it exists as a source in the wiki code); so in this case, I'd say it's actually better off with the original implementation...